pkg/agent/flowexporter/connections/conntrack.go

```diff
@@ -44,23 +44,25 @@ func filterAntreaConns(conns []*flowexporter.Connection, nodeConfig *config.Node
 		srcIP := conn.TupleOrig.SourceAddress
 		dstIP := conn.TupleReply.SourceAddress
 
-		// Only get Pod-to-Pod flows.
+		// Consider Pod-to-Pod, Pod-To-Service and Pod-To-External flows.
 		if srcIP.Equal(nodeConfig.GatewayConfig.IP) || dstIP.Equal(nodeConfig.GatewayConfig.IP) {
-			klog.V(4).Infof("Detected flow through gateway :%+v", conn)
+			klog.V(4).Infof("Detected flow through gateway %s :%+v", nodeConfig.GatewayConfig.IP.String(), conn)
 			continue
 		}
 
-		// Pod-to-Service flows w/ kube-proxy: There are two conntrack flows for every Pod-to-Service flow.
-		// One is with ClusterIP as source or destination, where other IP is podIP. Second conntrack flow is
-		// with resolved Endpoint Pod IP corresponding to ClusterIP. Both conntrack flows have same stats, which makes them duplicates.
-		// Ideally, we have to correlate these two connections and maintain one connection with both Endpoint Pod IP and ClusterIP.
-		// To do the correlation, we need ClusterIP-to-EndpointIP mapping info, which is not available at Agent.
-		// Therefore, we ignore the connection with ClusterIP and keep the connection with Endpoint Pod IP.
-		// Conntrack flows will be different for Pod-to-Service flows w/ Antrea-proxy. This implementation will be simpler, when the
-		// Antrea proxy is supported.
-		if serviceCIDR.Contains(srcIP) || serviceCIDR.Contains(dstIP) {
-			klog.V(4).Infof("Detected a flow with Cluster IP :%+v", conn)
-			continue
+		if !isAntreaProxy {
+			// Pod-to-Service flows with KubeProxy: There are two conntrack flows
```

Comment:
I always see it spelled "kube-proxy" in the docs (K8s docs and Antrea docs)

same comment for below